DataProcessor/DataUnpacker.h

```diff
@@ -113,10 +121,13 @@ public slots:
     bool mcu_check=false;
```

Comment:
On lines 118-120, can you remove `battery_eStop` and make the initial value of `door` be false? Also, the Q_PROPERTY for `battery_eStop` needs to be removed.